play-media-native/components/FacetFilters/FacetFilters.tsx

```diff
@@ -2,15 +2,20 @@ import { useCallback } from 'react';
 import { View } from 'react-native';
 
 import { FacetFilter, IFacetFilter } from './FacetFilter';
+import { CONTENT_TYPES } from '../../constants/contentTypes';
+import { useFilters } from '../../hooks/useFilters/useFilters';
 import { theme } from '../../theme/theme';
 import { DropdownItem } from '../DropdownPicker/DropdownPicker';
 
 type FacetFiltersProps = {
   facetFilters: IFacetFilter[];
   onChange: (id: string, item: DropdownItem) => void;
+  type: string;
 };
 
-export const FacetFilters = ({ facetFilters, onChange }: FacetFiltersProps) => {
+export const FacetFilters = ({ facetFilters, onChange, type }: FacetFiltersProps) => {
+  const { athleteFilterValues, eventFilterValues } = useFilters();
```

Comment:
Does a reusable component such as FacetFilters need to be connected to global state? What if we don't want to use global state with it (think using the filters without connecting it to a tab bar)? Why import it at all?

Pass parameters to reusable elements which you want to be flexible.

Reply:
The FacetFilters component serves a very specific purpose. I'm not against flexibility, but as the saying goes "premature optimization is the root of all evil".